### ISSUE TITLE
Storm Beast Large Statue

### ISSUE THREAD

**Issue by lancehalberd** [lancehalberd/spirit-world#32]
We have the sleeping storm beast but I would like statue scaled to the same size for the same pedestal in the material world. This is distinct from the smaller Storm Beast statue that appears as part of the War Temple boss. This should have similar design as the smaller statue though.

**Comment by Kaimatten** [comment]
Testing Comment and Attachment.

![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/92afea59-42f0-42b9-a69a-f3dd8c428ed6)

**Comment by Kaimatten** [comment]
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/928a38e9-aca3-4e8c-a947-72acd9260932)

Are you talking about having a sleeping storm beast statue but this size?

**Comment by lancehalberd** [comment]
No, the statue isn't sleeping, it is just located in the same place as the sleeping storm beast. 
The image you have pasted at 3x scale looks like it is sized about correctly for the boss fight (approximately 32px wide at the base).
The sleeping beasts are ~64px wide, so we would want the larger statues to be about double this width, although it wouldn't have to be proportionally as tall, it could potentially be a different pose if we wanted it to be more like 64x64 or 64x80. It also doesn't have to be the full 64px width, but on the scale so it is similar in scale to the sleeping beast.
Also, I want the pedestal that the statue is on to match the pedestal that the beast is sleeping on. This means either having a specific graphic shared for both or that I construct it from tiles.

**Comment by lancehalberd** [comment]
Here is what the pedestal looks like with the sleeping beast vs the placeholder statue graphics currently:

![Image](https://github.com/lancehalberd/spirit-world/assets/1253252/69a6c6d2-4d6b-4064-8bed-8b8ebaf1ac20)

![Image](https://github.com/lancehalberd/spirit-world/assets/1253252/53fe7256-302b-4d99-9c3e-bc6b982cc9e5)

The idea is you could use spirit sight while looking at the statues to see the actual beast sleeping in the spirit world. This would be repeated for the flame beast and frost beast on other platforms in the same room.

**Comment by Kaimatten** [comment]
I'll make a distinct 4 sided pedastal as well. I like the idea of design you have here with the lights and plain-ness, so I may keep a lot of that, but make the wall part distinct. It will end up the same amount of tiles, and will be big enough to hold the 3 different beasts and the statues.
The pose will probably be the same as the smaller statue, or at least similar. It should be something action looking.

**Comment by lancehalberd** [comment]
Sounds good. If you are adding a pedestal, please design it however you think would be best. The current design is just a quick temporary solution and doesn't really indicate too much other than the approximate size + height I was going for. The three statues are organized in a triangle with a large dais in the middle that holds the throne/altar for the War God. Just mentioning it in case the context informs the design at all and that the section for the War God's throne should work well near these as well.

**Comment by Kaimatten** [comment]
Honestly I'm liking the usage of the edge patterns here as it brings cohesiveness to other usages of the edge patterns across the world. I'll still do some changes to it, but it will mostly all be on the base of the pedastal.

**Comment by Kaimatten** [comment]
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/c187bceb-52f3-4b91-afcd-01ca42c56068)
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/1a1b6a7b-5a07-40c0-8bbd-757e42221eb9)
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/43ec1f4b-bafd-4b2e-930e-be8090af2dd0)

Real world and Spirit world pedastals, with a test to show the sleeping beast on it. Kept it the same sizing and kept the area for the sign to be put.
I'll probably a a bit of a magical feel/glow/effect to the spirit world pedestal as well. The goal is for it to feel like it is doing something magical constantly to the sleeping beast.

**Comment by lancehalberd** [comment]
This looks good to me. Are these ready for use or do you need to send final versions?

In general if things are in review and I'm happy with them, should I just comment to that effect, or how best to indicate that I've finished reviewing? I guess you are probably still planning on adding the statue graphic to this ticket as well right?

**Comment by Kaimatten** [comment]
Hm... We can either have it so you just comment that it is good and I take it from there, or we make another column of "ready for pngs" or something like that? I'm probably reading all the comments anyway so I don't know if it matters, but I guess the most logical is to have a column specifically that you reviewed it and it is clear for me to finish it up with all the animation notes/pngs.

Those two are ready to use, but you're right - I still need to make the Beast Statue for this card, so moving this back to High Priority.

**Comment by Kaimatten** [comment]
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/486440b1-6a41-47ca-825e-620104a66d24)

Statue on the material pedestal. If it's good I'll repost each part individually with no blank room on them.

**Comment by lancehalberd** [comment]
Cool, looks good to me!

**Comment by Kaimatten** [comment]
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/821130c2-ff95-4299-b4c8-893c9b1b580f)
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/00d7c280-d44c-4630-a88c-661d2e450299)
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/724cd622-d976-4af6-8c12-7a72c9328d98)
![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/86cc6b40-ad6b-49ce-bece-9ddbcf6fc408)

Pedestal1 is just the single frame of the Material world Pedestal we'll use for all 3 Beasts
SleepingStatue1 is the statue alone. I have it centered at the base of its foot against the pedestal, but it's fine however you match it up. The transparent shadow is build into the PNG

Pedestal2 is the animated glowing Pedestal for the Spirit World. The Gif gives example of what I imagined for it. The frames go 1-2-3-2-loop, and runs at 5 FPS.
Frame 1 - 1 second long
Frame 2 - .2 seconds
Frame 3 - 1 second long
Frame 2 - .2 seconds
Then loop.
Or whatever sort of like that. The sleeping Beast animation is the same as before, but I've attached it here again just in case.

![Image](https://github.com/lancehalberd/spirit-world/assets/141004606/b03fdf00-cbf1-4fc1-a941-9a11c2bacc70)
